Sources/SnapshotTesting/AssertInlineSnapshot.swift

```diff
@@ -13,13 +13,15 @@ import XCTest
 /// - Parameters:
 ///   - value: A value to compare against a reference.
 ///   - snapshotting: A strategy for serializing, deserializing, and comparing values.
+///   - recording: Whether or not to record a new reference.
 ///   - timeout: The amount of time a snapshot must be generated in.
 ///   - reference: The expected output of snapshotting.
 ///   - file: The file in which failure occurred. Defaults to the file name of the test case in which this function was called.
 ///   - line: The line number on which failure occurred. Defaults to the line number on which this function was called.
 public func _assertInlineSnapshot<Value>(
   matching value: @autoclosure () throws -> Value,
   as snapshotting: Snapshotting<Value, String>,
+  record recording: Bool = false,
```

Comment:
Do we need to add `record: recording` to the `_verifyInlineSnapshot` call in this method?

Reply:
Ha, good catch!